adage/analyze/tests.py

```diff
@@ -1025,3 +1034,106 @@ def test_expressionvalue_big_post(self):
             self.deserialize(resp)['meta']['total_count'],
             self.expressionvalue_count
         )
+
+
+@override_settings(HAYSTACK_CONNECTIONS=TEST_INDEX)
+class SearchIndexTestCase(ResourceTestCaseMixin, TestCase):
+    searchURI = '/api/v0/search/'
+    experiments = [
+        {
+            'accession': 'E-GEOD-31227',
+            'name': 'Expression data of Pseudomonas aeruginosa isolates from '
+                'Cystic Fibrosis patients in Denmark',
+            'description': 'CF patients suffer from chronic and recurrent '
+                'respiratory tract infections which eventually lead to lung '
+                'failure followed by death. Pseudomonas aeruginosa is one '
+                'of the major pathogens for CF patients and is the principal '
+                'cause of mortality and morbidity in CF patients. Once it '
+                'gets adapted, P. aeruginosa can persist for several decades '
+                'in the respiratory tracts of CF patients, overcoming host '
+                'defense mechanisms as well as intensive antibiotic '
+                'therapies. P. aeruginosa CF strains isolated from different '
+                'infection stage were selected for RNA extraction and '
+                'hybridization on Affymetrix microarrays. Two batch of P. '
+                'aeruginosa CF isolates are chosen : 1) isolates from a '
+                'group of patients since 1973-2008 as described in ref '
+                '(PMID: 21518885); 2) isolates from a group of newly '
+                'infected children as described in ref (PMID: 20406284).'
+        }, {
+            'accession': 'E-GEOD-24262',
+            'name': 'PA14_mexR vs. wildtype planktonic cells in minimal '
+                'medium with C-30',
+            'description': 'Mutations that made the cells insensitive to the '
+                'QS inhibition by C-30 were identified in mexR, a multi-drug '
+                'resistance operon repressor. Gene expression of mexR mutant '
+                'relative to wild-type at the presence of C-30 was examined. '
+                'Strains: PA14_mexR and wildtype. Medium: OS minimal medium + '
+                '0.1% Adenosine as carbon source. Compounds: 50 µM C-30 added '
+                'at OD600=0.25. Time: 2 hr. Temp: 37 ºC. Cell type: Planktonic '
```

Comment:
Hmm, Code Climate complains that this line is one character too long, which the linter on my text editor agrees with. Shall we move the trailing space to the next line?